Ginger/GingerCore/ALM/QtestCore.cs

```diff
@@ -0,0 +1,655 @@
+                                        {
+                                            if (step.CalledTestCaseId != null)
+                                            {
+                                                QTestApiModel.TestCaseWithCustomFieldResource calledTestCase = testcaseApi.GetTestCase((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), step.CalledTestCaseId);
+                                                foreach (QTestApiModel.TestStepResource nestedStep in calledTestCase.TestSteps)
+                                                {
+                                                    Activity matchingActivity = activities.Where(x => x.ExternalID == nestedStep.Id.ToString()).FirstOrDefault();
+                                                    if (matchingActivity != null)
+                                                    {
+                                                        QTestApiModel.TestStepLogResource testStepLog = new QTestApiModel.TestStepLogResource(null, nestedStep.Id);
+                                                        testStepLog.CalledTestCaseId = step.CalledTestCaseId;
+                                                        testStepLog.ParentTestStepId = step.Id;
+                                                        testStepLog.ActualResult = string.Empty;
+                                                        SetTestStepLogStatus(matchingActivity, ref testStepLog, statuses);
+                                                        stepsStatuses.Add(testStepLog.Status);
+                                                        testStepLogs.Add(testStepLog);
+                                                        testStepsCount++;
+                                                    }
+                                                }
+                                            }
+                                            else
+                                            {
+                                                Activity matchingActivity = activities.Where(x => x.ExternalID == step.Id.ToString()).FirstOrDefault();
+                                                if (matchingActivity != null)
+                                                {
+                                                    QTestApiModel.TestStepLogResource testStepLog = new QTestApiModel.TestStepLogResource(null, step.Id);
+                                                    testStepLog.ActualResult = string.Empty;
+                                                    SetTestStepLogStatus(matchingActivity, ref testStepLog, statuses);
+                                                    stepsStatuses.Add(testStepLog.Status);
+                                                    testStepLogs.Add(testStepLog);
+                                                    testStepsCount++;
+                                                }
+                                            };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)